src/cs/usfca/edu/histfavcheckout/filters/AnalyticsFilter.java

```diff
@@ -41,11 +41,10 @@ public void run() {
 				try {
 					boolean success = APIClient.sendEDR(edrRequest);
 					if(success) {
-						//System.out.println("EDR Event responded with non 200 responseCode: " + responseCode + " response!");
-						// LOG here EDR responded non 200 status and log what status was returned. 
+						System.out.println("EDR Event responded with non 200 responseCode: " + response + " response!");
 					}
 				} catch (IOException e) {
-					// LOG here that IOException occured when sending EDR Event.
+					System.out.println("IOException occured.");
```

Comment:
Please put the exception in the log. Also mention that the exception occurred while sending EDR.